assets/src/editor/events/tickets/editor-ticket/editor-ticket-entities-list.js

```diff
@@ -132,29 +153,17 @@ export default compose( [
 	),
 	withTicketPriceCalculatorFormModal,
 	withTicketEntityFormModal,
-	withDispatch(
-		( dispatch, { setNewTicketEntity, basePriceTypeEntity } ) => {
-			const { createEntity, createRelations } = dispatch( 'eventespresso/core' );
-			const addNewTicketEntity = async ( event ) => {
-				if ( event && event.preventDefault ) {
-					event.preventDefault();
-					event.stopPropagation();
-				}
-				const newTicket = await createEntity( 'ticket', {} );
-				const newBasePrice = await createEntity(
-					'price',
-					{ prtId: basePriceTypeEntity.id }
-				);
-				createRelations(
-					'ticket',
-					newTicket.id,
-					'prices',
-					[ newBasePrice ]
-				);
-				setNewTicketEntity( newTicket );
-			};
-			return { addNewTicketEntity };
-		}
-	),
+	createNewTicketEntity,
 	withPaginatedTicketEntitiesListAndFilterBar(),
 ] )( EditorTicketEntitiesList );
+
+/**
+ * a trimmed down ticket list: only includes ticket price calculator
+ */
+export const EditorTicketEntitiesOnlyList = compose( [
+	withPriceTypeEntities,
+	withNewTicketEntity,
+	withTicketPriceCalculatorFormModal,
+	withTicketEntityFormModal,
+	createNewTicketEntity,
+] )( EditorTicketEntitiesList );
```

Comment:
Would be helpful in the review to know why this is needed.

Reply:
I need to display a list of tickets in REM but don't want the full kit n kaboodle (ya whipper snapper)

Reply:
hmm, I'm wary of exporting components like this specifically for add-ons to use (what if another add-on needs something different?) and instead just export the individual parts so that other add-ons can build their own composed component as necessary?

Reply:
So with my previous comment, this is an example of something that could go in its own pull so we could discuss what api we want to expose for add-ons (third parties) to use here (and allow you to get the other changes in this pull merged quicker).

Reply:
or fix it later so I can continue on REM

Reply:
sure, will you create a card in the project/issue outlining what you did here, why you did it, and what's needed for REM?  I **did** see the inline comment with the code but:

- It doesn't explain _why_ this separate export, only _what_ it is. 
- It's not really accurate, it contains more than the ticket price calculator.
- As far as I can tell the only thing missing from the composition is the paginated filters?  If that's the case, then a possible option here is that the paginated filters are _added_ in somewhere else?  Maybe its an indicator that the structure of the components needs modified (if a common pattern/usage for this component will be without paginated filters?)

Anyways, I'm not going to comment on this further, your call what happens.